perses/app/setup_relative_calculation.py

```diff
@@ -74,6 +72,42 @@ def getSetupOptions(filename):
         if 'offline-freq' not in setup_options:
             setup_options['offline-freq'] = 10
             _logger.info(f"\t\t\toffline-freq not specified: default to 10.")
+    elif setup_options['fe_type'] == 'neq': #there are some neq attributes that are not used with the equilibrium samplers...
```

Comment:
Just for future reference, I think we could tidy up this bit of code that reads in the `.yaml` file, but I'll open an issue and add it to my todo list